### PR TITLE
Newest German changes, May 22nd

### PR DESCRIPTION
**Pull request by DonSqueak**

### CODE REVIEW

File: CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff [hunk 1]
```diff
@@ -1556,12 +1556,13 @@
...
       </trans-unit>
       <trans-unit id="You must have a subscription to use the app." xml:space="preserve">
-        <source>You must have a subscription to use the app.</source>
+        <source>You must have a subscription to use the app.</source><target>Du brauchst ein Abo, um die App zu benutzen.</target>
         <note>Shown in Settings as a subtitle if you haven't purchased the app</note>
       </trans-unit>
       <trans-unit id="Your %@ will expire on&#10;%@" xml:space="preserve">
         <source>Your %1$@ will expire on
-%2$@</source>
+%2$@</source><target>Dein %1$@ läuft ab am
```
Comment (DonSqueak) [inline CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:1564]: @cliss Would need to see it in context with the variables resolved to be sure if the grammatical gender is correct here...

Comment (DonSqueak) [reply CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:1564]: Meaning, what values can `%1$@` take?

**Comment by DonSqueak** [comment]
@technocidal fyi ;)

File: CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff [hunk 2]
```diff
@@ -209,6 +209,7 @@
       </trans-unit>
       <trans-unit id="Apple Keynote" xml:space="preserve">
         <source>Apple Keynote</source>
+        <target state="translated">Apple Keynote</target>
```
Comment (cliss) [inline CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:212]: Is this the same in German? Or did you miss this one by accident?

Comment (DonSqueak) [reply CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:212]: Left it as-is on purpose. The Apple Keynotes are known by this name, as in "Hast du die Apple Keynote gesehen?"

I would not expect anyone to say "Hast du die Apple-Präsentation gesehen"; especially not as a title for the thing. If we use the reference to the Apple Keynote, we best leave it as is, is my POV.

Comment (cliss) [reply CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:212]: 👍🏻 good deal, thanks. I assumed this wasn't an oversight, but I just wanted to double-check :)

**Review by cliss — APPROVED** [review]

File: CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff (hunk 2 above)
Comment (DonSqueak) [reply CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:212]: But I may be wrong here... @technocidal, what would you do here?

Comment (technocidal) [reply CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:212]: @DonSqueak I completely agree with your assessment. Keynote is well-known and pretty common even outside of tech circles. Lot's of German conferences use it to advertise their opening talk for example.

**Pull request merged**